src/submission/error-handler.service.ts

```diff
@@ -172,9 +172,11 @@ export class ErrorHandlerService {
       );
 
       // Email subject
-      const processCode = emailTemplateContextForUser.processCode || 'N/A';
-      const emailSubject = this.buildEmailSubject(processCode, emailTemplateContextForUser.orisCode, emailTemplateContextForUser.configuration);
-
+    const processCode = emailTemplateContextForUser?.processCode || 'N/A';
+    const env = this.configService.get<string>('app.env')?.trim()?.toLowerCase();
+    const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? ` (sent from ECMPS 2.0 ${env})` : '';
+    const emailSubject = `Error - ${processCode} Feedback for ORIS code ${emailTemplateContextForUser?.orisCode} Unit ${emailTemplateContextForUser?.configuration} ${subjectSuffix}`;
```

Comment:
This will not include the "(sent from ECMPS 2.0)" portion in production. Is that the intent, or do we only want to leave off the environment?
```suggestion
    const emailSubject = `Error - ${processCode} Feedback for ORIS code ${emailTemplateContextForUser?.orisCode} Unit ${emailTemplateContextForUser?.configuration} (sent from ECMPS 2.0${env && !['prod', 'production', ''].includes(env) ? ` ${env}` : ''})`;
```